cmake/presets/manual_selection.cmake

```diff
@@ -0,0 +1,69 @@
+set(PKG_ASPHERE OFF CACHE BOOL "" FORCE)
```

Comment:
Why are we using the `option()` syntax? And I don't think the `FORCE` argument is really needed.

Reply:
@junghans try it and come back to me. doesn't work without it. At least when I tested it.

Reply:
I did before asking. WIth a `CMakeLists.txt`:
```cmake
cmake_minimum_required(VERSION 2.8)
project(BLA LANGUAGES NONE)
set(BLA "Option1" CACHE STRING "BLA option")
message("BLA is ${BLA}")
```
and a `option2.cmake`:
```cmake
set(BLA "Option2" CACHE STRING "BLA option")
```
`cmake -C option2.cmake .` gives:
```
loading initial cache file ../option2.cmake
BLA is Option2
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/cmake/build
```
Am I missing something here?

Reply:
Yes, running it a second time (with a already set option) does not change it. I'm not just using it as "initial-cache". I'm forcing values with a preset.

Reply:
Ah!

Reply:
Ah!